deployment/workshop/keycloak/keycloak-operator-install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+for (( i=1 ; i <= $NUM_USERS ; i++ ))
```

Comment:
NUM_USERS is not set in the script so it gives me `(error token is " ")`.

Reply:
Sorry, the intent was that would have already been set by the previous step for creating the users and namespaces.